force_bdss/api.py

```diff
@@ -37,13 +37,13 @@
 from .mco.base_mco_factory import BaseMCOFactory  # noqa
 from .mco.i_evaluator import IEvaluator  # noqa
 from .mco.i_mco_factory import IMCOFactory  # noqa
+from .mco.optimizers.i_optimizer import IOptimizer  # noqa
 from .mco.parameters.base_mco_parameter_factory import BaseMCOParameterFactory  # noqa
 from .mco.parameters.base_mco_parameter import BaseMCOParameter  # noqa
 from .mco.parameters.mco_parameters import FixedMCOParameterFactory, RangedMCOParameterFactory, ListedMCOParameterFactory, CategoricalMCOParameterFactory, RangedVectorMCOParameterFactory  # noqa
 from .mco.parameters.mco_parameters import FixedMCOParameter, RangedMCOParameter, ListedMCOParameter, CategoricalMCOParameter, RangedVectorMCOParameter  # noqa
 from .mco.optimizer_engines.base_optimizer_engine import BaseOptimizerEngine  # noqa
 from .mco.optimizer_engines.weighted_optimizer_engine import WeightedOptimizerEngine  # noqa
-from .mco.optimizer_engines.scipy_optimizer_engine import ScipyOptimizerEngine  # noqa
```

Comment:
We could expose `ScipyOptimizer` here instead as it will be required by the ITWM plugin